modules/base-realms/realm-otp/idp-otp.tf

```diff
@@ -0,0 +1,51 @@
+resource "keycloak_oidc_identity_provider" "otp" {
+  realm = module.realm.id
+  alias = var.realm_name
+
+  enabled     = true
+  store_token = false
+  sync_mode   = "FORCE"
+
+  client_id     = var.client_id
+  client_secret = var.client_secret
+
+  first_broker_login_flow_alias = "first broker login"
+
+  default_scopes = "openid email"
```

Comment:
so we do get the email? Are we still hashing it for the sub or do we store it separately in the token?

Reply:
The hashed email is the sub and email comes under email claim.